go.mod

```diff
@@ -32,6 +33,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/hashicorp/go-metrics v0.5.2
 	github.com/rakyll/statik v0.1.7
+	github.com/regen-network/cosmos-proto v0.3.1 // TODO[1760]: Verify that this is still needed
```

Comment:
Once we get our proto-gen stuff updated, I don't think we'll need this anymore. Probably good to leave this comment there, but wanted to also answer it.